service/src/test/java/bio/terra/workspace/app/controller/ControlledGcpResourceApiControllerGcsBucketTest.java

```diff
@@ -100,6 +105,9 @@ public class ControlledGcpResourceApiControllerGcsBucketTest extends BaseConnect
               LifecycleAction.newDeleteAction(),
               LifecycleCondition.newBuilder().setAge(3).build()));
 
+  private static final String URL_LIST =
+      "https://raw.githubusercontent.com/yuhuyoyo/test-repo/main/helloworld.tsv";
```

Comment:
I think we will need these to be in a different place.

I also think we need some testing with real signed URLs. These are publicly available.